sequential/seq2pat.py

```diff
@@ -283,9 +293,27 @@ class Seq2Pat:
         sequences but no constraints are used to run the mining efficiently and practically.
         Power users can choose to drop this constraint by setting it to be None or increase the maximum span
         as the system has resources to support.
+    batch_size: Optional[int]
+        When batch_size is set, program runs Seq2Pat on batches of sequences instead of on the entire set of
+        sequences for improving scalability. batch_size defines the maximum number of sequences in a batch.
+    n_jobs: int
+        n_jobs defines the number of processes (n_jobs=2 by default) that are spawned
+        when mining tasks are applied among batches in parallel. If -1 all CPUs are used.
+    seed: int
+        Random seed to make sequences uniformly distributed among batches.
+    min_frequency_df: float
```

Comment:
I am not following what this parameter does.. also the name "df" is confusing, making the user thing it requires a dataframe.

Reply:
why do we need this parameter.. cannot we "auto" calculate sth using the other min_freq parameter and batch_size?

Reply:
The parameter tries to make `min_freq` to be a lower value for a batch, so the results from mining a batch will contain more patterns, to avoid a pattern that slightly falls below threshold gets dropped from batch, and thus dropped in the final results. I would think over how we can simply it.